templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}  
+  <h1>{{ object|yesno:"Update, Create" }} Manufacturer</h1>
+
+  <form action="" method="post" novalidate>
+  {% csrf_token %}
+    {{ form|crispy }}
+    <input class="btn btn-primary" type="submit" value="Submit">
+    <a class="btn btn-primary" href="{% url 'taxi:manufacturer-list' %}">Cancel</a>

```

Comment:
The 'Cancel' button should lead to the previous page the user was on, not a fixed URL. Consider using JavaScript to achieve this, such as `onclick="history.back()"`.